src/Sulu/Bundle/AdminBundle/Admin/Routing/FormOverlayListRouteBuilder.php

```diff
@@ -0,0 +1,222 @@
+
+        return $this;
+    }
+
+    public function setEditOverlayTitle(string $editOverlayTitle): FormOverlayListRouteBuilderInterface
+    {
+        $this->route->setOption('editOverlayTitle', $editOverlayTitle);
+
+        return $this;
+    }
+
+    public function setTabTitle(string $tabTitle): FormOverlayListRouteBuilderInterface
+    {
+        $this->route->setOption('tabTitle', $tabTitle);
+
+        return $this;
+    }
+
+    public function setTabOrder(int $tabOrder): FormOverlayListRouteBuilderInterface
+    {
+        $this->route->setOption('tabOrder', $tabOrder);
+
+        return $this;
+    }
+
+    public function setTabCondition(string $tabCondition): FormOverlayListRouteBuilderInterface
+    {
+        $this->route->setOption('tabCondition', $tabCondition);
+
+        return $this;
+    }
+
+    public function addListAdapters(array $listAdapters): FormOverlayListRouteBuilderInterface
+    {
+        $oldListAdapters = $this->route->getOption('adapters');
+        $newListAdapters = $oldListAdapters ? array_merge($oldListAdapters, $listAdapters) : $listAdapters;
+        $this->route->setOption('adapters', $newListAdapters);
+
+        return $this;
+    }
+
+    public function addLocales(array $locales): FormOverlayListRouteBuilderInterface
+    {
+        $oldLocales = $this->route->getOption('locales');
+        $newLocales = $oldLocales ? array_merge($oldLocales, $locales) : $locales;
+        $this->route->setOption('locales', $newLocales);
+
+        return $this;
+    }
+
+    public function setDefaultLocale(string $locale): FormOverlayListRouteBuilderInterface
+    {
+        $this->route->setAttributeDefault('locale', $locale);
+
+        return $this;
+    }
+
+    public function addToolbarActions(array $toolbarActions): FormOverlayListRouteBuilderInterface
+    {
+        $oldToolbarActions = $this->route->getOption('toolbarActions');
+        $newToolbarActions = $oldToolbarActions ? array_merge($oldToolbarActions, $toolbarActions) : $toolbarActions;
+        $this->route->setOption('toolbarActions', $newToolbarActions);
+
+        return $this;
+    }
+
+    public function setBackRoute(string $backRoute): FormOverlayListRouteBuilderInterface
+    {
+        $this->route->setOption('backRoute', $backRoute);
+
+        return $this;
+    }
+
+    public function enableSearching(): FormOverlayListRouteBuilderInterface
+    {
+        $this->route->setOption('searchable', true);
+
+        return $this;
+    }
+
+    public function disableSearching(): FormOverlayListRouteBuilderInterface
+    {
+        $this->route->setOption('searchable', false);
+
+        return $this;
+    }
+
+    public function enableMoving(): FormOverlayListRouteBuilderInterface
+    {
+        $this->route->setOption('movable', true);
+
+        return $this;
+    }
+
+    public function disableMoving(): FormOverlayListRouteBuilderInterface
+    {
+        $this->route->setOption('movable', false);
+
+        return $this;
+    }
```

Comment:
It seems like we've resolved a merge conflict in the wrong way :see_no_evil: This setting still exists in the `ListRouteBuilder`, although it is not used, because it has been removed in favour of the `toolbarActions`... Can you remove these functions in this and the other router builder?